File: JAVA/Greedy/src/등수매기기.java
import java.util.*;
public class 등수매기기 {
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        int arr[] = new int[N];
        for(int i=0; i<N; i++)
        {
            arr[i] = scanner.nextInt();
        }
        Arrays.sort(arr);
        long sum = 0;
        int abs;
        for(int i=0; i<N; i++)
        {
            abs = Math.abs(arr[i]-(i+1));
            sum += abs;
        }
        System.out.print(sum);
    }
}

File: Python/Greedy/등수 매기기.py
import  sys
input = sys.stdin.readline

n = int(input())

arr = []
for _ in range(n):
   arr.append(int(input()))

arr.sort()

hap = 0
for i in range(n):
    if arr[i] != i+1:
        hap += abs(arr[i] - (i+1))
    
print(hap)